src/DependencyInjection/Configuration.php

```diff
@@ -326,14 +326,18 @@ private function getExternalRedirectsNode(): ArrayNodeDefinition
         return $node;
     }
 
+    /**
+     * @deprecated since NelmioSecurityBundle 3.4.0, to be removed in 4.0.
+     */
```

Comment:
```suggestion
```
I think this is not needed, we deprecate the option, but we still need to call this method to add the configuration node